tests/WizardTest.php

```diff
@@ -142,3 +142,31 @@
 
     assertMatchesHtmlSnapshot($navigationHtml);
 });
+
+it('has the correct has step states', function () {
+    // Set up the step names array to match the expected steps
+    $stepNames = [
+        Livewire::getAlias(FirstStepComponent::class),
+        Livewire::getAlias(SecondStepComponent::class),
+        Livewire::getAlias(ThirdStepComponent::class),
+    ];
+
+    // Create instances of each step component and set the allStepNames property on them
+    $this->firstStep = new FirstStepComponent();
+    $this->firstStep->allStepNames = $stepNames;
+
+    $this->secondStep = new SecondStepComponent();
+    $this->secondStep->allStepNames = $stepNames;
+
+    $this->thirdStep = new ThirdStepComponent();
+    $this->thirdStep->allStepNames = $stepNames;
+
+    $this->assertFalse($this->firstStep->hasPreviousStep());
```

Comment:
Could you use Pest's expectation API instead of assertions?

Reply:
Tests have been updated 😄